src/main/java/lotto/model/LottoTicketGenerator.java

```diff
@@ -15,9 +20,20 @@ public class LottoTicketGenerator {
                 .collect(Collectors.toList());
     }
 
-    public static LottoTicket generate() {
-        Collections.shuffle(LOTTO_NUMBERS);
-        return LOTTO_NUMBERS.stream()
+    public static List<LottoTicket> generate(PurchaseAmount purchaseAmount) {
+        int count = (int) purchaseAmount.getPurchaseAmount() / LottoTicket.PRICE;
+        return generate(count);
+    }
+
+    public static List<LottoTicket> generate(int count) {
+        return Stream.generate(LottoTicketGenerator::generate)
+                .limit(count)
+                .collect(Collectors.toList());
+    }
+    private static LottoTicket generate() {
```

Comment:
메서드와 메서드 사이에 한 줄 공백을 주면 가독성이 높아질 것 같아요. 🙂

```suggestion
    }
    
    private static LottoTicket generate() {
```

Reply:
반영하였습니다.